packages/location-field/src/index.js

```diff
@@ -356,38 +418,51 @@ class LocationField extends Component {
       // Add the menu sub-heading (not a selectable item)
       // menuItems.push(<MenuItem header key='sr-header'>Search Results</MenuItem>)
 
+      // Split out transit results
```

Comment:
Also add comment about keeping the first three results.

Reply:
Should `3` become a prop or something?

Reply:
Good call! Comment and prop added in d030318b92928943de11c52377826231820d812d